_episodes/01-introduction.md

```diff
@@ -38,8 +38,7 @@ plt.show()  # Show the figure
 
 This code produces the following figure:
 
-<!-- ![basic_plot](https://matplotlib.org/stable/_images/sphx_glr_usage_002.png) -->
-![basic_plot](https://matplotlib.org/3.5.1/_images/sphx_glr_usage_001_2_0x.png)
+![basic_plot](https://matplotlib.org/3.5.1/_images/sphx_glr_usage_001_2_0x.png){:width="50%"}
```

Comment:
hmm, I'd imagine something like `max-width` would work better here. Else it will be very small when viewing the page in split-screen.

Reply:
Do you know what would be the syntax for `max-width` that would work here?

Reply:
yeah I wonder. Maybe the `{:width}` thing is arbitrary html attribute injection though?

Reply:
does `{:max-width=xxxpx}` work?

Reply:
I'm pretty sure we can inject classes, so worst case would be to define a CSS class and then assign that